### ISSUE TITLE
Error when trying to install vscode with snap

### ISSUE THREAD

**Issue by anderson-sousa** [tprasadtp/ubuntu-post-install#21]
First, I'd like to say that this is a very good project, really help me and saved me a lot of time.
Secondly, I try to install vscode with snap:

```yml
    snaps:
      classic:
        - vscode
```

and an error happened:

```bash
[sáb set 28 18:47:09 -03 2019] [     Info      ] Installig Snaps [Classic Packages]
[sáb set 28 18:47:09 -03 2019] [    Snap PKG   ] vscode...
[sáb set 28 18:47:10 -03 2019] [    Snap PKG   ] error: This revision of snap "vscode" was published using classic confinement and thus may perform
[sáb set 28 18:47:10 -03 2019] [    Snap PKG   ]        arbitrary system changes outside of the security sandbox that snaps are usually confined to,
[sáb set 28 18:47:10 -03 2019] [    Snap PKG   ]        which may put your system at risk.
[sáb set 28 18:47:10 -03 2019] [    Snap PKG   ] 
[sáb set 28 18:47:10 -03 2019] [    Snap PKG   ]        If you understand and want to proceed repeat the command including --classic.
[sáb set 28 18:47:10 -03 2019] [     Oh No!    ] Some Error occured while installing .
```

It seems that the snap option _--classic_ isn't been used to install the app. Is this a bug?

**Comment by tprasadtp** [comment]
Thank you,@anderson-sousa  
It indeed is a bug. It should be fixed now by #22 . Please report by testing it with `fix-snap-classic` branch.
![fix-snap](https://user-images.githubusercontent.com/10941447/65837588-8eadce00-e2f9-11e9-9b08-056541349f77.png)

**Comment by anderson-sousa** [comment]
I tested and it worked perfectly. Thanks!
